app/src/main/java/edu/uc/kovaciad/slicetracker/dao/IBrandDAO.kt

```diff
@@ -1,11 +1,9 @@
 package edu.uc.kovaciad.slicetracker.dao
 
 import androidx.lifecycle.LiveData
-import androidx.room.Dao
-import androidx.room.Delete
-import androidx.room.Insert
-import androidx.room.Query
+import androidx.room.*
```

Comment:
Avoid using the * wildcard import.  Specifically identifying what you are importing is better for documentation, and avoids clashes.  Example: java.util.* and java.sql.* but have a class named Date.  If you import both with wildcard, and then reference class Date, which one should it use?